app/javascript/components/admin/AdminTable.js

```diff
@@ -678,6 +678,7 @@ class AdminTable extends React.Component {
           selectAll={this.state.table.selectAll}
           entryOptions={this.state.entryOptions}
           entries={this.state.query.entries}
+          currentUser={this.props.current_user_email}
```

Comment:
Why was this added?

Reply:
used for a custom aria-label here: https://github.com/SaraAlert/SaraAlert/blob/508-compliance-updates/app/javascript/components/layout/CustomTable.js#L189

this covers the 6th bullet on the JIRA ticket